dsgrid/cli/registry.py

```diff
@@ -119,18 +119,24 @@ def list_(registry_manager):
 """
 
 
-@click.command(name="list")
+_list_dimensions_epilog = """
+Examples:\n
+$ dsgrid registry dimensions list\n
+$ dsgrid registry dimensions list -f "Type == sector"\n
+$ dsgrid registry dimensions list -f "Submitter == username"\n
+"""
+
+
+@click.command(name="list", epilog=_list_dimensions_epilog)
 @click.option(
     "-f",
     "--filter",
     multiple=True,
     type=str,
     help=f"""
-    filter table with a case-insensitive expression in the format 'field operation value',
-    accept multiple flags\b\n
+    Filter table with a case-insensitive expression in the format 'field operation value',
+    accepts multiple flags\b\n
```

Comment:
Should we also list "valid fields"?

Reply:
That's on the next line (ACCEPTED_OPS).

Reply:
Those are operators/operations. I'm wondering what I can apply the operators to. I.e., in "field operation value", we list the accepted operations, and given field and operation the acceptable values should be relatively obvious, but what fields are valid in this case? Is that information programmatically available here? If not, how does the user figure it out?

Reply:
The fields are the columns printed in the table. Presumably, the user will run the command at least once and display all rows.

Reply:
The fields are not available programmatically. They are hard-coded in each of the registry manager files.

Reply:
Got it. Would `column` or `table column` be more clear than `field`?